webapp/client/src/components/FormFieldCheckBox.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import checkedIcon from "../assets/icons/checked.svg";
+
+const CheckBox = styled.div`
+  & {
+    padding: 0;
+    margin-top: var(--spacing-02);
+    flex-wrap: inherit;
+  }
+
+  input {
+    width: 30px;
+    height: 30px;
+    opacity: 0;
+  }
+
+  input:focus + label {
+    box-shadow: 0 0 0 3px var(--focus-color);
+    background-color: var(--focus-color);
+  }
+
+  input:checked + label {
+    background-color: var(--link-color);
+    background-image: url(${checkedIcon});
+    background-repeat: no-repeat;
+    background-size: 22px;
```

Comment:
Probably also just copy+paste, but to make this more explicit and not rely on the order the labels have in the component, can we use something like `input:checked ~ label.checkmark`?

Reply:
Nice idea. I have changed it in the ConsentBox, too.